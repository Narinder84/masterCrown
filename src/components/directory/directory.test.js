/** @format */
import React from 'react';
import { shallow } from 'enzyme';
import DirectoryItem from './directory-item.componet';

it('it will see', () => {
	expect(shallow(<DirectoryItem />)).toMatchSnapshot();
});
