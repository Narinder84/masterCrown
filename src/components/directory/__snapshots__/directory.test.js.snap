// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it will see 1`] = `ShallowWrapper {}`;
